lib/second_page.dart

```diff
@@ -33,10 +30,34 @@ class SecondScreen extends StatelessWidget {
     ///Skapar TextField
     return Container(
       ///Drar in kanterna med 15 - ser snyggare ut
-      margin: EdgeInsets.only(left: 15, right: 15),
+      margin: const EdgeInsets.only(left: 15, right: 15),
       child: TextField(
-        decoration: InputDecoration(hintText: 'Task...'),
+        ///ger TextField en controller som kan spara värdet
+        controller: inputText,
+        decoration: const InputDecoration(hintText: 'Task...'),
       ),
     );
   }
+
+  Widget _addButton(context) {
+    return Row(
+      children: [
+        IconButton(
+          onPressed: () {
+            ///initierar variabel för varje sträng som skrivs in
+            var newTask = inputText.text;
+
+            ///provider ropar på addListItem och lägger till newTask
+            Provider.of<TodoListState>(context, listen: false)
```

Comment:
Saknar något form av villkor som kan hantera som användaren trycker på "Add" utan att skriva in text